### ISSUE TITLE
Predictions: wrong colors in inactive state

### ISSUE THREAD

**Issue by janezd** [biolab/orange3#5554]
**What's wrong?**

Selection background is black even when the widget has no keyboard focus.

**How can we reproduce the problem?** 

Open the widget and unfocus it.

This happens because the widget's table is composed of two tables. Only one can have focus, but both pretend to have it, otherwise one would have a grey selection. They do so by setting the state to active and has focus without checking whether the widget itself has focus or not.
